packages/eslint-plugin-pf-codemods/src/rules/v6/mastheadStructureChanges/masthead-structure-changes.test.ts

```diff
@@ -0,0 +1,165 @@
+const ruleTester = require("../../ruletester");
+import * as rule from "./masthead-structure-changes";
+
+ruleTester.run("masthead-structure-changes", rule, {
+  valid: [
+    {
+      code: `<Masthead  />`,
+    },
+    {
+      code: `import { Masthead } from '@patternfly/react-core'; <Masthead someOtherProp />`,
+    },
+  ],
```

Comment:
The valid tests should probably be updated. One would be the `code` from one of the invalid tests (without the import statement), and the other could be the `output` from one of the invalid tests.

Reply:
Whoops